README.md

```diff
@@ -38,33 +38,45 @@ vendor also provided simplified solution:
 |           | Measurement Register | Event Log      | Specification |
 | --------- | -------------------- | ---------      | ------------- |
 | vTPM      | TPM PCR              | TCG2 Event Log | [TPM2 Specification](https://trustedcomputinggroup.org/resource/tpm-library-specification/) </br> [TCG PC Client Platform TPM Profile Specification](https://trustedcomputinggroup.org/resource/pc-client-platform-tpm-profile-ptp-specification/) </br> [TCG PC Client Platform Firmware Profile Specification](https://trustedcomputinggroup.org/resource/pc-client-specific-platform-firmware-profile-specification/) |
-| Intel TDX | TDX MRTD/RTMR        | CC Event Log   | [Intel TDX Module 1.5 Base Architecture Specification](https://cdrdv2.intel.com/v1/dl/getContent/733575) </br> [Intel TDX Virtual Firmware Design Guide](https://cdrdv2.intel.com/v1/dl/getContent/733585) </br> [td-shim specification](https://github.com/confidential-containers/td-shim/blob/main/doc/tdshim_spec.md) |
+| Intel TDX | TDX MRTD/RTMR        | CC Event Log   | [Intel® TDX Module 1.5 Base Architecture Specification](https://cdrdv2.intel.com/v1/dl/getContent/733575) </br> [Intel® TDX Virtual Firmware Design Guide](https://cdrdv2.intel.com/v1/dl/getContent/733585) </br> [td-shim specification](https://github.com/confidential-containers/td-shim/blob/main/doc/tdshim_spec.md) |
 
 ![](docs/cc-trusted-foundation.png)
 
-## 3. SDKs
+## 3. APIs
 
-The back-end SDK or service will produce the TCB measurements:
+CC Trusted APIs aims to collect confidential primitives (i.e., measurement, event log, quote) for zero-trust design, supporting multiple deploy environments (firmware/VM/cloud native cluster).
+The [APIs](common/python/cctrusted_base/api.py) are designed to be vendor agnostic and TCG compliant APIs. The APIs will keep evolving on demand. 
+
+| API | Description  | Parameters  | Response  |
+| --- | ------------- |----- |----- |
+| get_default_algorithms | Get the default Digest algorithms supported by trusted foundation. | | The default algorithms |
+| get_measurement_count | Get the count of measurement register. | | The count of measurement registers |
+| get_measurement | Get measurement register according to given selected index and algorithms. | imr_select ([int, int]): The first is index of measurement register, the second is the algorithms ID | The count of measurement registers |
+| get_quote | Get the quote for given nonce and data. | | The `Quote` object |
+| get_eventlog | Get eventlog for given index and count. | | `TcgEventLog` object |
```

Comment:
get_eventlog has two optional inputs, which are 'start' and 'count'. Start is the index of the event log to start fetching. Count is the number of event logs to fetch.